examples/README.md

```diff
@@ -228,28 +236,36 @@ This script will read messages from the terminal and pass it to the topic. From
  
 You should now be able to see this data entry in the KV bucket `test-data`.
 
+Sources [StreamingKVExample.scala](./src/main/scala/com/basho/riak/spark/examples/streaming/StreamingKVExample.scala)
+
 ### Spark Streaming Riak TS Example
 
 Having seen how Spark Streaming works with KV buckets, let's now look at the TS table example.
 
-This example will start a stream from the Kafka topic `streaming` into the TS table `ts_weather_demo`. This stream will run until terminated. Whenever a message is produced for Kafka topic `streaming`, the Spark Streaming context will automatically stream the message from the topic into the TS table. To see this in action, we first need to create and activate the TS table. You can find more information on creating and activating TS tables [here](http://docs.basho.com/riak/ts/latest/using/creating-activating/). For this demo we will create and activate the table with the following:
+This example will start a stream from the Kafka topic `ingest-ts` into the TS table `ts_weather_demo`. This stream will run until terminated. Whenever a message is produced for Kafka topic `ingest-ts`, the Spark Streaming context will automatically stream the message from the topic into the TS table. To see this in action, we first need to create and activate the TS table. You can find more information on creating and activating TS tables [here](http://docs.basho.com/riak/ts/latest/using/creating-activating/). For this demo we will create and activate the table with the following:
 
 ```
 riak-admin bucket-type create ts_weather_demo '{"props":{"n_val":3, "table_def": "CREATE TABLE ts_weather_demo (weather VARCHAR NOT NULL, family VARCHAR NOT NULL, time TIMESTAMP NOT NULL, temperature DOUBLE, humidity DOUBLE, pressure DOUBLE, PRIMARY KEY ((weather, family, QUANTUM(time, 1, 'h')), weather, family, time))"}}'
 
 riak-admin bucket-type activate ts_weather_demo
 ```
 
-Now that we have created a TS table, you can run the `StreamingTSExample.scala` example with:
+Now that we have created a TS table, we need to create the `ingest-ts` topic:
+
+```
+path/to/kafka/bin/kafka-topics.sh --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic ingest-ts
+```
+
+Now, we can run the `StreamingTSExample.scala` example with:
 
 ```
 /path/to/spark-riak-connector-examples/bin/run-example streaming.StreamingTSExample
```

Comment:
Please shorten `/path/to/spark-riak-connector-examples/bin/run-example` to `bin/run-example` so it matches the other examples, as mentioned above.